relay-server/src/actors/processor.rs

```diff
@@ -2439,6 +2439,94 @@ mod tests {
 
     use super::*;
 
+    /// Ensures that if we ratelimit one batch of buckets in [`EncodeMetrics`] message, it won't
+    /// also ratelimit the next batches in the same message automatically.
+    #[cfg(feature = "processing")]
+    #[tokio::test]
+    async fn test_ratelimit_per_batch() {
+        use relay_metrics::BucketValue;
+        use relay_quotas::{Quota, ReasonCode};
+        use relay_test::mock_service;
+        use std::sync::atomic::AtomicUsize;
+        use std::sync::atomic::Ordering::SeqCst;
```

Comment:
Is there a reason to not move these imports to the tests module?

Reply:
they're just used in processing mode so in non-processing i'd get unused_import warnings. could have put them outside and under a feature flag too of course, I thought both ways are fine

Reply:
We have some tests that only run in processing mode. Adding a test for processing relays is alright.

Reply:
The code with a `#[cfg(feature = "processing")]` flag, including imports and tests, is only available in processing relays. A test with that flag that uses imports with also that flag should result in no unused import warnings, unless I'm missing something. It's possible to group imports with the `cfg` with something like the following:
```rs
#[cfg(feature = "processing")]
use {
  module_a::A;
  module_b::B;
}
```

Reply:
ok yeah, thats what I was thinking about. As I said i thought both ways are fine but i did it the way you showed now